json-smart/src/main/java/net/minidev/json/parser/JSONParser.java

```diff
@@ -93,6 +93,13 @@ public class JSONParser {
 	 * @since 2.4
 	 */
 	public final static int BIG_DIGIT_UNRESTRICTED = 2048;
+
+	/**
+	 * If limit the max depth of json size
+	 *
+	 * @since 2.5
```

Comment:
2.5... I lose the count 😄

Reply:
@UrielCh I apologize, but I didn't quite understand what you meant. Could you please explain it again?

Reply:
lol,
@since should refer to the first version number that contains this code.

so the should be the version number in pom.xml for the next version.

Reply:
2.5 is the the correct next version number, nice.